src/diagnostic-channel-publishers/src/mongodb-core.pub.ts

```diff
@@ -35,7 +35,7 @@ const mongodbcorePatchFunction: PatchFunction = function(originalMongoCore) {
 };
 
 export const mongoCore2: IModulePatcher = {
-    versionSpecifier: ">= 2.0.0 < 2.2.0",
+    versionSpecifier: ">= 2.0.0 < 4.0.0",

```

Comment:
This doesn't really match the name of the patch :)

Reply:
renamed to remove the 2